bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -199,6 +223,10 @@
 		<label>Wifi Enabled</label>
 		<description>Indicates whether the wifi network is enabled</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Status</tag>
```

Comment:
Why status while this channel allow enabling or disabling with ?

Reply:
Changed to Switch.Enabled